filter.json

```diff
@@ -25,5 +25,12 @@
   "treadeoffer",
   "discordgifts",
   "discorcl",
-  "dlscord.\n"
+  "dlscord.\n",
+  "Discord Nitro",
+  "Nitro",
+  "bit.ly",
+  "for Free",
```

Comment:
Note: these should be case insensitive but that's a further improvement

Reply:
This can cause false triggering. For example, members can talk about discord nitro and get banned for "scam".

Reply:
Okay, that's valid. In our server we don't talk in English so it wouldn't false trigger.